packages/cli/src/utils.ts

```diff
@@ -283,7 +283,7 @@ const getAndroidDetails = async (
   const minSdk = new RegExp(
     AaptPrefixes.sdkPrefix + AaptPrefixes.quoteRegex
   ).exec(stdout);
-  const permissions = [...stdout.matchAll(/uses-permission: name='(.*)'/g)];
+  const permissions = [...stdout.matchAll(/uses-permission: name='(.+)?' maxSdkVersion='(\d\d)'|uses-permission: name='(.+)'/g)];
```

Comment:
`name='(.+)?'`

Won't the `?` serve to make the characters between the `''` optional? To make this match stingy, I think it should be `name='(.+?)'`.